src/os/bluestore/BlueStore.h

```diff
@@ -1450,6 +1450,15 @@ class BlueStore : public ObjectStore,
     b->shared_blob->bc.write(txc->seq, offset, bl, flags);
     txc->shared_blobs_written.insert(b->shared_blob);
   }
+
+  template <typename T, typename F>
+  T select_option(CollectionRef c, const std::string& opt_name, T val1, F f) {
```

Comment:
opt_name is needed for the future per-pool vs. per-store setting selection. CollectionRef is redundant.